src/mixins/renderer.js

```diff
@@ -1,15 +1,31 @@
 import Parser from 'htmlparser2/lib/Parser'
 
 export default function(incrementalDom) {
-  const {
-    attr,
-    elementClose,
-    elementOpen,
-    elementOpenEnd,
-    elementOpenStart,
-    elementVoid,
-    text,
-  } = incrementalDom
+  const autoClosingStack = []
+
+  const autoClosing = () => {
+    const stack = autoClosingStack.shift()
+    if (!stack) return
+
+    stack.reverse().forEach(tag => incrementalDom.elementClose(tag))
```

Comment:
This change means that the elements correction by htmlparser2 can no longer use. So we added the logic to close orphan elements caused by passing invalid HTML.